build_tools/bazel/iree_check_test.bzl

```diff
@@ -104,11 +111,16 @@ def iree_check_single_backend_test_suite(
           if opt_flags is specified.
       opt_flags: If specified, source files are preprocessed with OPT_TOOL with
           these flags.
+      target_cpu_features: currently unimplemented (must be empty), will eventually allow specifying target CPU features.
       tags: tags to apply to the generated tests. Note that as in standard test suites, manual
           is treated specially and will also apply to the test suite itself.
       timeout: timeout for the generated tests.
       **kwargs: any additional attributes to pass to the underlying tests and test suite.
     """
+
+    if target_cpu_features:
+        fail("target_cpu_features must currently be empty")
```

Comment:
iiuc, at the moment we're never targeting aarch64 in bazel builds. so the list of legal values here is currently an empty list (since the only legal value "dotprod" is aarch64-specific).